### PR TITLE
Add few another get methods into Transaction

### PR DESCRIPTION
**Pull request by wmdmgv**

### CODE REVIEW

File: src/Entities/Transaction.php [hunk 1]
```diff
@@ -69,6 +69,14 @@ public function getCreatedTime()
     /**
      * @return string
      */
+    public function getUpdatedTime()
```
Comment (slavcodev) [inline src/Entities/Transaction.php]: This method already exists in the parent class

File: src/Entities/Transaction.php [hunk 2]
```diff
@@ -193,4 +201,84 @@ public function getInvoiceIds()
...
+     */
+    public function isHas3ds()
+    {
+        return $this->getAttribute('has3ds');
+    }
+
+    /**
+     * @return bool
+     */
+    public function isHasDcc()
```
Comment (2e3s) [inline src/Entities/Transaction.php]: probably `is` is excessive

Comment (wmdmgv) [inline src/Entities/Transaction.php]: but what about isApproved (bool) method?

Comment (2e3s) [inline src/Entities/Transaction.php]: "is approved" sounds fine, "is has dcc" sounds ridiculous.

**Pull request merged**